server/base/src/main/java/org/apache/accumulo/server/util/checkCommand/UserFilesCheckRunner.java

```diff
@@ -20,18 +20,33 @@
 
 import static org.apache.accumulo.server.util.Admin.CheckCommand.Check;
 
+import org.apache.accumulo.core.metadata.AccumuloTable;
+import org.apache.accumulo.server.ServerContext;
+import org.apache.accumulo.server.cli.ServerUtilOpts;
 import org.apache.accumulo.server.util.Admin;
+import org.apache.accumulo.server.util.RemoveEntriesForMissingFiles;
 
 public class UserFilesCheckRunner implements CheckRunner {
   private static final Check check = Check.USER_FILES;
 
   @Override
-  public Admin.CheckCommand.CheckStatus runCheck() {
+  public Admin.CheckCommand.CheckStatus runCheck(ServerContext context, ServerUtilOpts opts,
+      boolean fixFiles) throws Exception {
     Admin.CheckCommand.CheckStatus status = Admin.CheckCommand.CheckStatus.OK;
+    printRunning();
 
-    System.out.println("Running check " + check);
-    // work
-    System.out.println("Check " + check + " completed with status " + status);
+    System.out.println("\n********** Looking for missing user files **********\n");
```

Comment:
These prints produce a lot of noise when running the command.  Could make these log at trace/debug level. This would make the output of the command very concise when there are no problems. For testing purpose could change the log4j2 testing config to set the check logger to trace/debug.  

When a problem is found could log the details of that at warn.  I rant this locally and manually corrupted the metadata table and saw the following output.

```
********** Looking for missing columns **********

Scanning the accumulo.metadata (!0) table for missing required columns...
Tablet +scanref;m is missing required columns: col FQs: [srv:dir, ~tab:~pr], col fams: [] in the accumulo.metadata (!0) table
Tablet +scanref< is missing required columns: col FQs: [srv:dir], col fams: [] in the accumulo.metadata (!0) table

********** Looking for invalid columns **********

```

So maybe in that output everything could be logged at trace/debug and the two lines about problems fouind at WARN level.

If this seems like a good change, it could  be done in a follow on PR.

Reply:
Thanks, this is a nice way to avoid the verbose output. The change made sense to make in this PR, so I made it.